Meetme.Client/src/App.tsx

```diff
@@ -1,18 +1,92 @@
-import { useState } from "react";
+import { useEffect } from "react";
 import { Route, Routes } from "react-router-dom";
 import Navbar from "./components/Navbar/Navbar";
 import FindPair from "./components/FindPair/FindPair";
 import Profile from "./components/Profile/Profile";
+import Callback from "./components/Callback/Callback";
+import { useDispatch, useSelector } from "react-redux";
+import { RootState } from "./store/authReducer";
+import { login, logout } from "./store/authSlice";
 
 export default function App() {
-	const [isAuthenticated, setIsAuthenticated] = useState(false);
+	const dispatch = useDispatch();
+	const isAuthenticated = useSelector(
+		(state: RootState) => state.auth.isAuthenticated
+	);
+
+	useEffect(() => {
+		const checkAuth = async () => {
+			try {
+				const response = await fetch("http://localhost:5041/auth/status", {
+					method: "GET",
```

Comment:
"http://localhost:5041/auth/status" -> http://localhost:5041/ to env, auth/status to constants